packages/lib-auth/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@zooniverse/auth",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "axios": "~0.19.0",
+    "cookie": "~0.4.0",
+    "detect-node": "~2.0.4",
+    "yup": "~0.27.0"
+  },
+  "scripts": {
+    "docs": "jsdoc src",
```

Comment:
Where do the docs get published? Should they be included with this PR?

Reply:
They're not currently published - they could be, but I didn't have time to work out where the best place to do that is / how to automate it when I was finishing up. Have you got any ideas?

Reply:
How about `docs/` in the package root? Maybe we generate them manually, for the time being, until we figure out automating them. We could perhaps generate them when pushing to GitHub.

Reply:
[`jsdoc2md`](https://www.npmjs.com/package/jsdoc-to-markdown) looks useful since `jsdoc` produces HTML by default.

Reply:
Voilà, [a generated README](https://github.com/zooniverse/front-end-monorepo/tree/fb511c23363b174e89092b6624cbfc02a0a0fbc5/packages/lib-auth/docs). If I install `jsdoc2md` globally, then we can use this to replace the handwritten Readme files in the classifier.

Reply:
With an appropriate ADR, of course.